From 5ef63eefd55d45d2fb1f9e1a20e42f1163121132 Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Thu, 1 Aug 2013 13:41:57 +0800
Subject: [PATCH 1/2] ENGR00273437 only build LegacyCamera on mx6sl

only build LegacyCamera on mx6sl, there is
no GPU 3D core on mx6sl so use LegacyCamera
instead of 3D Camera, on other chips which
has GPU 3D core LegacyCamera will not build.

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Android.mk b/Android.mk
index ec46e82..ec552da 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,3 +1,4 @@
+ifneq ($(HAVE_FSL_IMX_GPU3D),true)
 LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
@@ -20,3 +21,4 @@ ifeq ($(strip $(LOCAL_PACKAGE_OVERRIDES)),)
 # Use the following include to make our test apk.
 include $(call all-makefiles-under, $(LOCAL_PATH))
 endif
+endif
-- 
1.8.0

